<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchReferenceWidget</class>
 <widget class="QWidget" name="BatchReferenceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch Reference Download</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="4" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Error</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPlainTextEdit" name="plainError"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Empty</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPlainTextEdit" name="plainEmpty"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushSearchMissing">
     <property name="text">
      <string>Search Missing</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="format">
      <string>%v of %m</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Completed</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainCompleted"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Unsupported</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="plainUnsupported"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="labelCurrent">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
